src/beliefs/k_previous_observations.jl

```diff
@@ -0,0 +1,21 @@
+
+struct KMarkovUpdater <: Updater
+    k::Int64
+end
+
+function initialize_belief{O}(bu::KMarkovUpdater, obs::O, b=nothing)

```

Comment:
We should get rid of the `b=nothing`. That's not part of POMDPs.jl anymore and I think it will only confuse people.